frontend/src/components/cluster/Overview.tsx

```diff
@@ -135,13 +128,14 @@ function EventsSection() {
                 interactive
               >
                 <Box>{makeStatusLabel(event)}</Box>
-              </LightTooltip>,
+              </LightTooltip>
           },
         ]
           :
           []
         }
-        data={events ? events.sort(sortFunc) : null}
+        data={events}
+        defaultSortingColumn={2}
```

Comment:
I thought we were going to use the idea that the `defaultSortingColumn` is indexed from 1 so we can use negative values for decreasing sorting without having a 2nd property passed.